src/EntryPoints/Specials/SpecialManageApprovers.php

```diff
@@ -46,7 +47,7 @@ private function isAdmin(): bool {
 	}
 
 	/**
-	 * @param array $approversCategories
```

Comment:
When typehinting arrays the type must be specified. Otherwise this comment serves no purpose.

Reply:
Yes. I know that, it must have missed my eye since they were generated by IDE.